src/main/java/net/openhft/chronicle/network/cluster/handlers/CspTcpHandler.java

```diff
@@ -43,8 +46,10 @@
     protected final List<WritableSubHandler<T>> writers = new CopyOnWriteArrayList<>();
 
     @NotNull
-    private final Map<Long, SubHandler<T>> cidToHandle = new HashMap<>();
+    private final TLongObjectMap<SubHandler<T>> cidToHandle = new TLongObjectHashMap<>();
```

Comment:
Use LongObjectMap to reduce boxing garbage

Reply:
nice one